Sources/Examples/Echo/Runtime/main.swift

```diff
@@ -98,7 +101,9 @@ struct Echo: ParsableCommand {
         try! client.channel.close().wait()
       }
 
-      callRPC(self.rpc, using: client, message: self.message)
+      for _ in 0..<self.iterations {
```

Comment:
I think the formatter is gonna get mad at you, I think you want:

```swift
for _ in 0 ..< self.iterations {
```

Or you can run `./scripts/format.sh`